### ISSUE TITLE
Application crashes after enabling - disabling video

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#31]
```
What steps will reproduce the problem?
1. During a call enabling video works
2. I want to disable it so I just click the "Disable video" menu-button
3. The application crashes (I get "The application IMSDroid (process 
org.doubango.imsdroid) has stopped unexpectedly")

What version of the product are you using? On what operating system?
Android 2.1 - htc g1
```

Original issue reported on code.google.com by `sylar1...@gmail.com` on 12 Aug 2010 at 10:21

**Comment by GoogleCodeExporter** [comment]
```
same with 2.1 on Samsung Galaxy Spica, crash also happens when you end the 
video call
```

Original comment by `volkov.r...@gmail.com` on 12 Aug 2010 at 2:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 12 Aug 2010 at 4:26
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Can someone test this patch?
```

Original comment by `boss...@yahoo.fr` on 12 Aug 2010 at 5:18

Attachments:
- [VideoProducer.patch](https://storage.googleapis.com/google-code-attachments/imsdroid/issue-31/comment-3/VideoProducer.patch)

**Comment by GoogleCodeExporter** [comment]
```
I tested your patch and it is working on my HTC EVO.
```

Original comment by `whg...@gmail.com` on 13 Aug 2010 at 8:00

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.263.
```

Original comment by `boss...@yahoo.fr` on 16 Aug 2010 at 2:12
- Changed state: **Fixed**
